README.md

```diff
@@ -44,6 +44,26 @@ app.import('node_modules/showdown-highlight/lib/index.js', {
 app.import('node_modules/highlight.js/styles/tomorrow-night.css');
 ```
 
+There may be cases where you need additional rollup plugins in order to import your CommonJs files. For example, you may need to handle builtin node require statements, i.e.
+
+```
+var path = require('path');
+```
+
+For this, you may want to import the rollup plugin `rollup-plugin-node-builtins`. In order to do this, just include the plugin in your `package.json` and use the following syntax in your `ember-cli-build.js` or `index.js`:
+
+```js
+const nodeBuiltins = require('rollup-plugin-node-builtins');
+
+app.import('node_modules/some-name/index.js', {
+  using: [
+    { transformation: 'cjs', as: 'some-name', plugins: [ nodeBuiltins() ]}
+  ]
+});
+```
+
+You can include multiple rollup plugins into the array and pass in options via the arguments to your plugin creation function.
```

Comment:
Shouldn't it be "you can include multiple plugins *in* the array"?

Reply:
Yep. @Gaurav0 with those two minor tweaks, looks good to me 👍

Reply:
Thanks!